package/rpm/SOURCES/usr/sth/conf/README.md

```diff
@@ -0,0 +1,31 @@
+# Short Term History configuration procedure
+
+STH is able to start multiple instances by adding and configuring certain files of this directory
+using [sth](../../../etc/init.d/sth "sth") service script
+
+To start multiple instances is needed to place in `/usr/sth/conf` one configuration file for 
+each instance that is wanted to run. RPM cames with one preconfigured instance (config 
+file called sth_default.conf) that can be used as a template to configure another instances.
```

Comment:
`sth_default.conf` could be renamed as `sth_default.conf.template`, or simply `sth.conf.template`.